api/config-dev.yaml

```diff
@@ -65,9 +65,3 @@ MLPConfig:
 TuringUIConfig:
   ServingDirectory: ../ui/build
   ServingPath: /turing
-OpenapiConfig:
-  ValidationEnabled: true
-  SpecFile: api/openapi.yaml
-  SwaggerUIConfig:
-    ServingDirectory: api/swagger-ui-dist
-    ServingPath: /api-docs/
```

Comment:
What's the motivation for removing it?

Reply:
1. It is already set as the defaults and an example of how to override them is written in api/config/example.yaml. I would think that this config-dev.yaml is more of a minimal way of running them.
2. If the api/swagger-ui-dist is not generated, the api will error out because it selects which API validation on the following rule; if this config is not set, use the handwritten openapi yaml spec.